tests/CommunityToolkit.Mvvm.UnitTests/Test_ObservablePropertyAttribute.cs

```diff
@@ -1062,7 +1153,38 @@ public BroadcastingViewModelWithInheritedAttribute(IMessenger messenger)
         }
 
         [ObservableProperty]
-        [AlsoBroadcastChange]
+        [NotifyRecipients]
+        private string? name2;
+    }
+
+    [NotifyRecipients]
+    partial class BroadcastingViewModelWithClassLevelAttribute : ObservableRecipient
+    {
+        public BroadcastingViewModelWithClassLevelAttribute(IMessenger messenger)
+            : base(messenger)
+        {
+        }
+
+        [ObservableProperty]
+        private string? name;
+    }
+
+    [ObservableRecipient]
+    [NotifyRecipients]
+    partial class BroadcastingViewModelWithAttributeAndClassLevelAttribute : ObservableObject
```

Comment:
may want to add a test with VM inheriting from this class too like `BroadcastingViewModelWithInheritedAttributeAndClassLevelAttribute`

Reply:
Good idea, added in 901d23bb06465d929b3518add458ff540002adfe. Thanks! 😄